### PR TITLE
CM-175:Blazor WebAssembly Ahead of Time Compilation (AoT)

### PR DESCRIPTION
**Pull request by cjaynjoku**

### CODE REVIEW

**Comment by asif-anwar** [comment]
retest this please

**Comment by asif-anwar** [comment]
retest this please

**Comment by CodeMazeBlog** [comment]
Retest this please

**Comment by asif-anwar** [comment]
retest this please

**Comment by asif-anwar** [comment]
retest this please

**Comment by asif-anwar** [comment]
retest this please

**Comment by asif-anwar** [comment]
retest this please

**Comment by jshergal** [comment]
@asif-anwar - the build is failing as there is a workload that needs to be installed in the CI build. I have reached out to the CodeMaze support to get that issue addressed. Sorry for the trouble.

File: blazor-features/BlazorWebAssemblyAheadOfTimeCompilation/Tests/PrimeNumberComponentTest.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            Services.AddSingleton(mockPrimeNumberService.Object);
+
+            var cut = RenderComponent<PrimeNumber>();
+            var timeout = TimeSpan.FromSeconds(5);
+            cut.WaitForAssertion(() => Assert.NotNull(cut.Instance.PrimeNumbers), timeout);
+
+            Assert.IsType<List<int>>(cut.Instance.PrimeNumbers);
+            Assert.Equal(new List<int> { 2, 3, 5, 7, 11 }, cut.Instance.PrimeNumbers);
+        }
+    }
```
Comment (karimkod) [inline blazor-features/BlazorWebAssemblyAheadOfTimeCompilation/Tests/PrimeNumberComponentTest.cs:22]: @cjaynjoku can you check if this test is succeeding locally in your machine?

**Review by karimkod — CHANGES_REQUESTED** [review]

**Pull request closed without merging**